logs_intake/socket/SendLogEntriesOverSocket.py

```diff
@@ -0,0 +1,104 @@
+# This code is provided as an example only.
+import json
+import time
+import socket
+import ssl
+
+ddApiKey = "<your_api_key>"
+
+# metadata: Additional metadata to send with the logs
+metadata = {
+    "ddsourcecategory": "aws",
+}
+
+host = "intake.logs.datadoghq.com"
+ssl_port = 10516
+
+DD_SOURCE = "ddsource"
+DD_CUSTOM_TAGS = "ddtags"
+
+metadata[DD_SOURCE] = "apigee"
+metadata[DD_CUSTOM_TAGS] = "level:info,x.api:identity-v1,env:dev,project:apigee"
+
+
+def lambda_handler(event):
+    # Check prerequisites
+    if ddApiKey == "<your_api_key>" or ddApiKey == "":
```

Comment:
the user has already defined `ddApiKey` up top, why are we having them define it again here?

Reply:
It's a test, not a definition.  As this code may be used by someone else, we need to check that the value has been provided - otherwise when they run the code and get success but no data appearing in DD, it's confusing.  This way if they haven't set the key a few lines earlier, they get a useful error message.